examples/build.gradle

```diff
@@ -14,6 +16,8 @@ dependencies {
     implementation 'org.junit.jupiter:junit-jupiter:5.8.2'
     implementation 'com.madgag.spongycastle:core:1.51.0.0'
     implementation 'com.madgag.spongycastle:prov:1.51.0.0'
+
+
```

Comment:
Generally, when making PRs, please look at your own PR and what the reviewer will see. The first impression is: 18 files change. Try to either functionally change files or not touch them at all. Or if you want to change or fix the formatting, fix it for all files.

```suggestion
```